### PR TITLE
attempt to fix grammar for ArgumentList and CoverParenthesizedExpressionAndArrowParameterList

### PR DESCRIPTION
**Pull request by michaelficarra**
I haven't rendered this yet to try it out because I have no idea how to do that.

### CODE REVIEW

**Comment by jeffmo** [comment]
1. `npm install`
2.  run `node node_modules/ecmarkup/bin/ecmarkup.js spec/index.htm index.htm --css ecmarkup.css --js ecmarkup.js`
3. Open up index.htm in your browser

**Comment by michaelficarra** [comment]
I figured it out. Fixing the problems now...

**Comment by michaelficarra** [comment]
Okay, take a look.

File: spec/index.htm [hunk 1]
```diff
@@ -68,77 +68,28 @@
     <h1>|Arguments|</h1>
 
     <emu-production name="Arguments" params="Yield">
-      <emu-rhs>()</emu-rhs>
+      <emu-rhs>( )</emu-rhs>
       <emu-rhs>(<emu-nt params="?Yield">ArgumentList</emu-nt>)</emu-rhs>
       <ins class="block">
-        <emu-rhs>(<emu-nt params="?Yield">ArgumentList</emu-nt>,)</emu-rhs>
-        <emu-rhs a="Arguments--ArgumentList-AssignmentExpression">(<emu-nt params="?Yield">ArgumentList</emu-nt>, ...<emu-nt params="In, ?Yield">AssignmentExpression</emu-nt>)</emu-rhs>
```
Comment (jeffmo) [inline spec/index.htm:75]: Deleting this and the refactoring of `ArgumentList` below allows for a comma after rest params, which shouldn't be allowed

Comment (jeffmo) [inline spec/index.htm:75]: Oh nvm, these are spreads not rests -- so trailing comma is fine (per offline convo).

**Pull request merged**